Tests/WebSocketKitTests/PMCETests.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  File.swift
+//
+//
+//  Created by Jimmy Hough Jr on 5/17/23.
+//
+
+import XCTest
+@testable import WebSocketKit
+
+class PMCEConfigTests:XCTestCase {
+    typealias Config = PMCE.DeflateConfig
+   
+    
+}
+
+// design tests
+class PMCETests:XCTestCase {
+   
```

Comment:
This really has to be filled in

Reply:
I might remove it as I HAD tests that sendable refactor made irrelevant. They were all just commented out but I nuked them as the refactor changes all the add to be invalid tests really. Im not sure of the value the would have really as just having tests isnt the same as having useful tests that actually tell you something. 
I will ponder it and try to add something at least.